features/poller/monitors/core/src/main/java/org/opennms/netmgt/poller/monitors/HostResourceSwRunMonitor.java

```diff
@@ -198,7 +197,7 @@ public void rowCompleted(SnmpRowResult result) {
                 final SnmpValue name = nameResults.get(nameInstance);
                 final SnmpValue value = statusResults.get(nameInstance);
                 // See if the service name is in the list of running services
-                if (name != null && value != null && match(serviceName, stripExtraQuotes(name.toString()))) {
+                if (name != null && value != null && match(serviceName, StringUtils.stripExtraQuotes(name.toString()))) {
```

Comment:
I guess this regex correctly fixes NMS-3882 then, since strings with internal quotes would have been hacked up by the antlr.StringUtils call. In any case, the regex is the behavior that is intended.